### PR TITLE
Replaced choices for models

### PR DESCRIPTION
**Pull request by juampivallejo**
TODO: Test carefully and front integration.

### CODE REVIEW

File: api/tests/test_winery.py [hunk 1]
```diff
@@ -179,15 +185,17 @@ def test_wine_detail_get(self):
 
     def test_varietal_get(self):
         response = self.client.get(
-            reverse('varietals')
+            reverse('varietals-list')
         )
-        expected = [{'id': k, 'value': v} for k, v in VARIETALS]
+        expected = [{'id': self.varietal.id, 'name': self.varietal.name}]
```
Comment (eliasjuanpablo) [inline api/tests/test_winery.py]: We should keep the same keys so we don't break the current frontend implementation.

File: users/admin.py [hunk 2]
```diff
@@ -1,4 +1,6 @@
 from django.contrib import admin
+from django.contrib.auth.models import Permission
 from .models import WineUser
 
 admin.site.register(WineUser)
+admin.site.register(Permission)
```
Comment (eliasjuanpablo) [inline users/admin.py]: :eyes:

Comment (juampivallejo) [reply users/admin.py]: should i delete it?

Comment (juampivallejo) [reply users/admin.py]: I removed it for more consistency

**Pull request merged**